torch/fx/symbolic_trace.py

```diff
@@ -87,6 +88,8 @@ def __enter__(self):
     def __exit__(self, type, value, tb):
         sys.setprofile(None)
 
+HOLE = object()
```

Comment:
The name `HOLE` seems kinda weird, maybe something like `PLACEHOLDER` or `PH` to align with the terminology in graph?